Automattic/ElasticPress
# Updating fork to 10up's 3.5.6

## Description

This PR merges the changes to the upstream Elasticpress repository, so we are aligned in their 3.5.6 release. 

As a bonus, we now get GitHub Actions-based tests, which do work and pass. Also, we've made some minor tweaks at our script to run the tests locally.

*Please test this changes consciously, this PR might potentially break things*.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR in your mu-plugins environment (local or sandbox).
2. Test that search-related functionalities work.
